src/pages/index.js

```diff
@@ -1,72 +1,143 @@
 import "./index.css";
-import { initialCards, selectors, constants } from "../utils/constants.js";
+import {
+  selectors,
+  constants,
+  validationSettings,
+} from "../utils/constants.js";
 import Card from "../components/Card.js";
 import FormValidator from "../components/FormVaildator.js";
 import Section from "../components/Section.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import UserInfo from "../components/UserInfo.js";
 import PopupWithImage from "../components/PopupWithImage.js";
+import Api from "../components/Api.js";
+import PopupWithConfirmation from "../components/PopupWithConfirmation.js";
 
 // ! ||--------------------------------------------------------------------------------||
 // ! ||                              class instantiations                              ||
 // ! ||--------------------------------------------------------------------------------||
 
-const cardSection = new Section(
-  {
-    items: initialCards,
-    renderer: renderCard,
-  },
-  selectors.cardSection
-);
-
-cardSection.renderItems(initialCards);
-
 const cardAddPopupWithForm = new PopupWithForm("#card-add", (values) => {
   const newTitle = values.title;
   const newLink = values.description;
   const newCardData = {
     name: newTitle,
     link: newLink,
   };
-  renderCard(newCardData);
-  constants.cardAddForm.reset();
-  addFormVaildator.disableSubmitButton();
+  cardAddPopupWithForm._submitBtn.textContent = "Saving...";
+  api
+    .addCard(newCardData)
+    .then((data) => {
+      renderCard(data);
+      cardAddPopupWithForm.close();
+      constants.cardAddForm.reset();
+      addFormVaildator.disableSubmitButton();
+    })
+    .catch((err) => {
+      console.error(err);
+      alert(`${err}, Could not add new card`);
+    })
+    .finally(() => {
+      cardAddPopupWithForm._submitBtn.textContent = "Save";
+    });
 });
 
 cardAddPopupWithForm.setEventListeners();
 
+const userInfo = new UserInfo({
+  nameSelector: ".profile__title",
+  jobSelector: ".profile__description",
+  avatarSelector: "#profile-avatar",
+});
+
 const profileEditSubmitPopupWithForm = new PopupWithForm(
   "#profile-edit",
   (values) => {
-    constants.profileTitle.textContent = values.title;
-    constants.profileDescription.textContent = values.description;
-    profileEditSubmitPopupWithForm.close();
+    profileEditSubmitPopupWithForm._submitBtn.textContent = "Saving...";
```

Comment:
`_submitBtn`


Please, delete the **low dash** (`underscore`) from methods if you want to use them `outside` a class because the `low dash` shows a **private** method which is used only inside the class.


**Please, correct it everywhere**